### ISSUE TITLE
Simpler format for the CtrlPBufTag window

### ISSUE THREAD

**Issue by jeromedalbert** [ctrlpvim/ctrlp.vim#223]
When I use `:CtrlPBufTag` for just one file, I get this output:

<img width="668" alt="ctrlpbuftag" src="https://cloud.githubusercontent.com/assets/236461/14061622/e92c4844-f341-11e5-9e5c-3ce3296ca1a8.png">

It is a bit noisy compared to the traditional CtrlP file window. Being only in one file, the file name (in blue here) is redundant. Is there a way to display only the white part? (or the orange part)

**Comment by tacahiroy** [comment]
Indeed, it's redundant for `:CtrlPBufTag`, but not for `:CtrlPBufTagAll`

**Comment by jeromedalbert** [comment]
Yeah I understand that.

I am using `:CtrlPBufTag` not only as a fuzzy searcher but also as a file tag outliner so that I don't need an additional plugin like [tagbar](https://github.com/majutsushi/tagbar). When looking at the general outline of the file, it's nice when there is less noise.

Just saying!
